packages/clerk-js/src/utils/captcha/turnstile.ts

```diff
@@ -58,6 +58,23 @@ interface RenderOptions {
    * @default 'always'
    */
   appearance?: 'always' | 'execute' | 'interaction-only';
+  /**
+   * The widget theme. Can take the following values: light, dark, auto.
+   * The default is auto, which respects the user preference. This can be forced to light or dark by setting the theme accordingly.
```

Comment:
```suggestion
   * The default is auto, which respects the user preference. This can be forced to light or dark by setting the theme accordingly.
   * @default 'auto'
```